planning/robot_model_loader/src/robot_model_loader.cpp

```diff
@@ -51,11 +51,12 @@ robot_model_loader::RobotModelLoader::RobotModelLoader(const Options &opt)
   configure(opt);
 }
 
-robot_model_loader::RobotModelLoader::~RobotModelLoader()
-{
+robot_model_loader::RobotModelLoader::~RobotModelLoader() {
   model_.reset();
   rdf_loader_.reset();
   kinematics_loader_.reset();
+
+  ROS_DEBUG( "Robot model loader destructor" );
```

Comment:
If you think all these debug statements are needed, can you namespace them - ROS_DEBUG_NAMED("robot_model_loader", ...) so that I can hide them as needed?